Makefile

```diff
@@ -136,6 +136,7 @@ test: all
 
 lint:
 	flake8 src test tools pyodide_build benchmark
+:
```

Comment:
Hi @rth , I'm not totally sure but this colon disables the clang-format call, is this wanted?

Reply:
Thanks @phorward ! That was a typo in earlier versions..

Reply:
hehe yeah I've checked this right after I commented ;-) had too many tabs open.